mos-platform/atari8-xegs/syms.s

```diff
@@ -0,0 +1,13 @@
+; xegs ROM size, only 32, 64, 128 and 256 are valid sizes (in KiB)
+
+.weak __cart_rom_size
+; default to 256K
+__cart_rom_size = 256
+
+; explicitly select bank 0, allows for override of INIT
+.text
+	.global _cart_init
```

Comment:
Does it make sense for a user to provide a different version of this? If so, the symbol should be weak.

Reply:
Is there a reason not to allow the user to define a different _cart_init? That would be worth writing down here as a comment if there is one.

Reply:
Yes.